modules/social_features/social_featured_content/templates/group--featured.html.twig

```diff
@@ -63,7 +67,8 @@
 
     {% if content.field_group_location|render or content.field_group_address|render %}
       <div class="teaser__content-line">
-        <svg class="teaser__content-type-icon" aria-hidden="true" title="{% trans %}Event location{% endtrans %}">
+        <svg class="teaser__content-type-icon">
```

Comment:
This SVG should be hidden because there's an sr-only line with the same information right after it.

Reply:
Fixed